src/api/v1/permissions.json

```diff
@@ -91,9 +91,7 @@
       "user_details-own:delete",
       "constants-all:read"
     ],
-    "ORG_NON_ADMIN": [
-      "user_details-child:write"
-    ],
+    "ORG_NON_ADMIN": [],
```

Comment:
Can you remove this Set from `database/utils/permissions.js`

Reply:
I guess we might as well just remove all references to `ORG_NON_ADMIN` in this PR?